plugins/frontend/backstage-plugin-cloudsmith/src/api/CloudsmithClient.ts

```diff
@@ -53,4 +54,56 @@ export class CloudsmithClient implements CloudsmithApi {
     }
     return await response.json();
   }
+
+  // get data from Cloudsmith quota endpoint (https://help.cloudsmith.io/reference/quota_read)
+  async getQuota({ owner }: { owner: string }): Promise<any> {
+    const cloudsmithApiUrl = await this.getApiUrl();
+    const response = await this.fetchApi.fetch(
+      `${cloudsmithApiUrl}/quota/${owner}/`,
+    );
+    if (!response.ok) {
+      throw new Error(`Failed to retrieve quota: ${response.statusText}`);
+    }
+    return await response.json();
+  }
+
+  // get repository audit logs from Cloudsmith audit repo logs endpoint (https://help.cloudsmith.io/reference/audit_log_repo_list) and support pagniation
+
+  async getRepoAuditLogs({
+    owner,
+    repo,
+  }: {
+    owner: string;
+    repo: string;
+  }): Promise<any> {
```

Comment:
You should create a new `RepoAuditLogs` type for the response and return the following from this function.

Promise<RepoAuthLogs>